src/factory/cosmosCDCService.ts

```diff
@@ -81,14 +82,15 @@ export const cosmosCDCService = {
               pipe(
                 leaseContainer,
                 O.fold(
-                  () => createContainer(database, LEASE_CONTAINER_NAME),
+                  () =>
+                    createContainer(database as Database, LEASE_CONTAINER_NAME),
                   (lContainer) => TE.right(lContainer),
                 ),
                 TE.chain((lContainer) =>
                   processChangeFeed(
-                    container,
+                    container as Container,
```

Comment:
do we need explicit cast here?

Reply:
Yes, because processChangeFeed accepts a Container, while the interface accepts a Resource that is defined as Collection | Container

Reply:
The same also for the other casts